src/frontend/src/observer/observer_manager.rs

```diff
@@ -58,6 +60,7 @@ impl ObserverNodeImpl for FrontendObserverNode {
             Info::User(_) => {
                 self.handle_user_notification(resp);
             }
+            Info::Mapping(_) => self.handle_table_mapping_notification(resp),
```

Comment:
Suggest using a more concrete name for `Mapping` or `table_mapping`, like `vnode_mapping`.